ncm-pam/src/main/pan/components/pam/config.pan

```diff
@@ -54,11 +54,16 @@ function pam_add = {
 		ret[service][pamtype] = list();
 	};
 	tail = length(ret[service][pamtype]);
-        options = nlist();
-        if (exists(ARGV[4])) {
-                options = ARGV[4];
+    options = nlist();
+    options_list = null;
+    if (exists(ARGV[4])) {
```

Comment:
can you add some annotation to the function describing the existing an dthe new argument